src/test/java/core/basesyntax/service/amount/BalanceHandlerTest.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.amount;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BalanceHandlerTest {
+    private static final FruitTransaction bananaBalance;
+    private final OperationHandler balance = new BalanceHandler();
+
```

Comment:
What will this test return without settings to bananaBalance any values?
Maybe try to invoke get on empty Storage.fruits
.... and in other handler tests too